### ISSUE TITLE
Get ongoing response?

### ISSUE THREAD

**Issue by YukunYangNPF** [Jaseibert/QualtricsAPI#8]
Hi. I just want to know if the get_survey_responses supports getting partial ongoing responses, the responses in progress that have not been closed.

**Comment by Tyler-Owen-Morris** [comment]
No, it does not currently get ongoing responses, but that could be added as a config parameter quite easily.

**Comment by Tyler-Owen-Morris** [comment]
yes, it does support this. You just need to pass the function the key word argument "exportResponsesInProgress=True" and it will export partial responses in your data.
